src/main/webapp/app/overview/exercise-chatbot/exercise-chatwidget/exercise-chat-widget.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component } from '@angular/core';
+import { faPaperPlane } from '@fortawesome/free-solid-svg-icons';
+import { faCompress } from '@fortawesome/free-solid-svg-icons';
+import { faXmark } from '@fortawesome/free-solid-svg-icons';
+import { MatDialog } from '@angular/material/dialog';
+
+@Component({
+    selector: 'jhi-exercise-chat-widget',
+    templateUrl: './exercise-chat-widget.component.html',
+    styleUrls: ['./exercise-chat-widget.component.scss'],
+})
+export class ExerciseChatWidgetComponent {
+    messages: string[] = [];
+    IrisMessages: string[] = ['Hey! How can I help you?'];
```

Comment:
Make sure to use proper code styling.
```suggestion
    irisMessages: string[] = ['Hey! How can I help you?'];
```